### PR TITLE
Add web link properties from API

### LINKED ISSUE #9: Fix account slug in weblink
Cobalt asset web links are prefixed with their account like `https://app.cobalt.io/{{ accountSlug }}`. The JupiterOne integration takes a naive approach to building this `accountSlug` property:

https://github.com/JupiterOne/graph-cobalt/blob/master/src/steps/account.ts#L27
```ts
  await apiClient.getAccount(async (org) => {
    delete org.resource.token;
    const orgProps = org.resource;
    const webLink: string = `https://app.cobalt.io/${webifyFromTitles(
      orgProps.name,
    )}`;
```

The `org.resource.name` can be converted into `accountSlug` in some cases, but is unreliable. JupiterOne could allow users to pass an optional `accountSlug` property into the integration that can allow reliable web links to be created.

The Cobalt API does not currently return web URLs or the `accountSlug` property, but if this is available in their APIs in the future, we should leverage Cobalt's own API for more reliable web links.

### PR DESCRIPTION
**Pull request by ndowmon**
Fixes #9

### CODE REVIEW

File: src/steps/__snapshots__/index.test.ts.snap [hunk 1]
```diff
@@ -127,7 +129,7 @@ Object {
       "description": "",
       "displayName": "Test Cloud Config",
       "name": "Test Cloud Config",
-      "webLink": "https://app.cobalt.io/test-org-api/assets/as_HrZ74Uh",
+      "webLink": "undefined/assets/as_HrZ74Uh",
```
Comment (aiwilliams) [inline src/steps/__snapshots__/index.test.ts.snap]: Is this okay?

Comment (ndowmon) [reply src/steps/__snapshots__/index.test.ts.snap]: @aiwilliams updated. Good catch

**Review by austinkelleher — DISMISSED** [review]

**Review by aiwilliams — DISMISSED** [review]

**Review by austinkelleher — APPROVED** [review]

**Comment by nktrk** [comment]
Long time @ndowmon @aiwilliams was checking in on J1 and saw this wasn't merged yet. Anything I can help with?

**Comment by ndowmon** [comment]
Hey @nktrk thanks for the bump! Just seem to have lost track of this one after versioning.

**Pull request merged**